TypeScript support

Howarto opened the pull request:
Added a TypeScript file definition to support TypeScript language.

catc commented:
Hey, thanks for doing this. Just a few things:
1. Can you add the type definition for the newly added `ignoreFn` option? see https://github.com/catc/displace/pull/17
1. Can you check how relevant the windows installation stuff still is after the last PR added `cross-env`? See https://github.com/catc/displace/pull/17/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R7
1. If you rebase and fix any merge conflicts I can merge it

Howarto commented:
> 
> 
> Hey, thanks for doing this. Just a few things:
> 
>     1. Can you add the type definition for the newly added `ignoreFn` option? see #17
> 
>     2. Can you check how relevant the windows installation stuff still is after the last PR added `cross-env`? See https://github.com/catc/displace/pull/17/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R7
> 
>     3. If you rebase and fix any merge conflicts I can merge it

Hey!

1. Added the new type definition.
2. cross-env fixes Windows problems. So I removed my changes in README file.
3. I did a rebase and all the conflicts are fixed.

Also, I rebuilt the project to include the PR changes that you accepted relative to preventDefault call.

catc commented:
Looks great! thanks @Howarto
